python/ray/serve/tests/test_logging.py

```diff
@@ -306,21 +310,28 @@ def check_log():
         class_method_replica_id = resp2["replica"].split("#")[-1]
         if json_log_format:
             user_method_log_regex = (
-                f'.*"deployment": "{resp["app_name"]}_fn", '
+                ".*"
+                f'"actor_id": "{resp["actor_id"]}", '
+                f'"worker_id": "{resp["worker_id"]}", '
+                f'"deployment": "{resp["app_name"]}_fn", '
                 f'"replica": "{method_replica_id}", '
                 f'"component_name": "replica", '
+                '"message":.* user func.*, '
                 f'"request_id": "{resp["request_id"]}", '
                 f'"route": "{resp["route"]}", '
-                f'"application": "{resp["app_name"]}", "message":.* user func.*'
```

Comment:
why are the message fields removed?

Reply:
It's just reordered to before the request id. This is due to my refactor for drying up the `ADD_IF_EXIST_FIELDS` thing. If we do want message to go last I can do move it down. Just feel since it's structured json, the order shouldn't matter

Reply:
nope doesn't matter